vlayer/ethereum/circuits/lib/src/slot_test.nr

```diff
@@ -36,6 +36,23 @@ mod mapping {
     }
 }
 
+mod mapping_by_uint {
+    use crate::slot::mapping_by_uint;
+
+    #[test]
+    fn uint_mapping() {
+        let slot = [
+            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 10
```

Comment:
Can we somehow generate this value from [10] or 10?

Reply:
change proposed:
```
let slot = field_to_bytes32(10);
```